supply-chain/audits.toml

```diff
@@ -899,6 +899,12 @@ criteria = "safe-to-deploy"
 delta = "0.15.1 -> 0.16.0"
 notes = "The primary change here is the switch from the `hdwallet` dependency to using `bip32`."
 
+[[audits.zcash_primitives]]
```

Comment:
Nit: we generally do not add audits of our own Zcash code, because it would be "self-auditing" mixed in with our third-party auditing. Instead we mark that we trust our own crates (which is a correct position to convey to external consumers of our crates).

Reply:
Okay; we can fix this up in a later commit.